Share The Load/Screens/ViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  ViewController.swift
+//  Share The Load
+//
+//  Created by Christopher Hefferman on 10/01/2021.
+//
+
+import UIKit
+import Firebase
+import RealmSwift
+
+class ViewController: UIViewController {
+    
+    private let getRealmQuestionsButton = UIButton()
+    private let removeFromRealmButton = UIButton()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        fetchQuiz()
+        createGetRealmQuestionsButton()
```

Comment:
For setting up constraint based layouts you can use loadView() instead of viewDidLoad, it's worth having a look at where that fits in the VC lifecycle, we use it quite a lot on confirmed

Reply:
Good to know - I've not used loadView() before, so i've implemented this now. Let me know if the implementation of it is 👍🏼 or 👎🏼 on my next PR.